rdr_service/dao/study_nph_dao.py

```diff
@@ -474,6 +500,63 @@ def __init__(self):
     def get_id(self, obj: OrderedSample):
         return obj.id
 
+    @staticmethod
+    def _is_order_cancelled(order: Order) -> bool:
+        return order.status == "cancelled"
+
+    @staticmethod
+    def _is_ordered_sample_cancelled(ordered_sample: OrderedSample) -> bool:
+        return ordered_sample.status == "cancelled"
```

Comment:
Might need to cast these as with `lower` since there is no rules on how this is getting stored in the model.